scalding-parquet/src/main/scala/com/twitter/scalding/parquet/tuple/scheme/TypedParquetTupleScheme.scala

```diff
@@ -140,14 +140,14 @@ class TypedParquetTupleScheme[T](val readSupport: ParquetReadSupport[T], val wri
   type SourceCallType = SourceCall[Array[AnyRef], Reader]
   type SinkCallType = SinkCall[Array[AnyRef], Output]
 
-  override def sourceConfInit(flowProcess: FlowProcess[JobConf], tap: TapType, jobConf: JobConf): Unit = {
+  override def sourceConfInit(flowProcess: FlowProcess[_ <: JobConf], tap: TapType, jobConf: JobConf): Unit = {
```

Comment:
JobConf is the old hadoop1 api and Configuration is hadoop2. In cascading3, Hfs taps and descendents were moved to the hadoop2 api. Cascading's MultiInputFormat still looks at the hadoop1 key for the input formats though (see https://github.com/twitter/elephant-bird/pull/465). So broadly, the changes involved moving anything that's closely tied to the Hfs tap to use Configuration, leaving the remaining to use JobConf, and still using the Deprecated wrappers for the EB input and output formats.